docs/source/quickstart.rst

```diff
@@ -78,7 +78,7 @@ run remotely in the background.
 Gather
 ~~~~~~
 
-The ``map/submit`` functions return ``Future`` objects, lightweight tokens that
+The ``map/submit`` functions return ``Future`` objects: lightweight tokens that
```

Comment:
I think that a comma is still appropriate here.

Reply:
I can't remember the in's and out's of "Eats, Shoots & Leaves" by Lynne Truss. But I think you are right. "," for small pauses, ";" for medium pauses and ":' for longer pauses.